packages/flutter/test/material/popup_menu_test.dart

```diff
@@ -1779,6 +1780,161 @@ void main() {
       lessThan(600 - windowPaddingBottom), // Device height is 600.
     );
   });
+
+  group('feedback', () {
+    late FeedbackTester feedback;
+
+    setUp(() {
+      feedback = FeedbackTester();
+    });
+
+    tearDown(() {
+      feedback.dispose();
+    });
+
+    testWidgets('PopupMenuButton with enabled feedback', (WidgetTester tester) async {
+      const bool enableFeedback = true;
+
+      await tester.pumpWidget(
+        MaterialApp(
+          home: Scaffold(
+            body: PopupMenuButton<dynamic>(
```

Comment:
PopupMenuButton<int> 

Same comment for the tests below

Reply:
The tester fails to find the element when using ```PopupMenuButton``` instead of ```PopupMenuButton<dynamic>```. How may I fix this?